example/gluon/sn-gan/README.md

```diff
@@ -0,0 +1,40 @@
+# Spectral Normalization GAN
```

Comment:
Please update  https://github.com/apache/incubator-mxnet/tree/master/example#deep-learning-examples-in-the-mxnet-project-repository with your example, thanks!